src/leaderboard.js

```diff
@@ -0,0 +1,23 @@
+const { redis } = require('./redis');
+
+const addTeam = (org, score, name) => {
+    redis.zadd([org, score, name])
+        .then(console.log)
+        .catch(console.error);
+}
+
+const updateTeamScore = (org, score, name) => {
+    redis.zincrby([org, score, name])
+        .then(console.log)
+        .then(console.error)
+}
+
+const getSortedTeamsByScore = (org) => {
+    redis.zrevrange([org, 0, -1])
+        .then(console.log)
+        .then(console.error)
+}
+
```

Comment:
Instead of just logging the results, can they be returned? To make that syntactically cleaner and easier to see, try to write your functions using the async-await patterns instead of with `.then()`.

Also, address Michael's commend about testing. Are there unit tests that we can run for these? If not, how can they be created or how can we test if this is working?

Reply:
First point done.

We cannot write unit tests for these because it requires integration with a db. However, these have been verified manually, and we can write integration tests in the future once an environment is set up.